Dockerfile

```diff
@@ -473,16 +473,14 @@ ARG CGO_CXXFLAGS="${CXXFLAGS}"
 ARG CGO_LDFLAGS="${LDFLAGS}"
 
 WORKDIR /home/builder/sdk-go/src
-RUN ./make.bash --no-clean
+RUN ./make.bash
 
 # Build the standard library with and without PIE. Target binaries
 # should use PIE, but any host binaries generated during the build
 # might not.
 WORKDIR /home/builder/sdk-go
-ENV GOOS="${GOOS}" \
-  GOROOT="/home/builder/sdk-go" \
-  GOPATH="/home/builder/gopath" \
```

Comment:
This block was making things slightly more complex and ended up not really adding anything. Everything behaved normally when I removed it.